devsetup/scripts/gen-edpm-baremetal-kustomize.sh

```diff
@@ -123,5 +120,32 @@ cat <<EOF >>kustomization.yaml
         namespace: ${NAMESPACE}
 EOF
 fi
+if [ "$EDPM_PROVISIONING_INTERFACE" != "" ]; then
```

Comment:
This is not optional when BMO deployed with CRC, unlike CBO and it's not in the sample spec[1]

[1] https://github.com/openstack-k8s-operators/dataplane-operator/blob/main/config/samples/dataplane_v1beta1_openstackdataplanenodeset_baremetal_with_ipam.yaml#L21